VideoLocker/src/main/java/org/edx/mobile/view/WebViewFindCoursesActivity.java

```diff
@@ -69,9 +76,54 @@ public boolean onQueryTextChange(String newText) {
                 return false;
             }
         });
+        searchView.setOnQueryTextFocusChangeListener(new View.OnFocusChangeListener() {
+            @Override
+            public void onFocusChange(View view, boolean queryTextFocused) {
+                if (!queryTextFocused) {
+                    searchActive = false;
+                    searchItem.collapseActionView();
```

Comment:
`collapseActionView()` doesn't seem to be what we want here, since that's for collapsing menu items visible in the action bar to the overflow menu. In any case, this only seems to work if the `showAsAction` attribute is set to `collapseActionView`.